src/components/OcBreadcrumb.vue

```diff
@@ -81,16 +97,19 @@ export default {
     },
   },
   computed: {
-    $_ocBreadcrumb_dropdownItems() {
+    dropdownItems() {
       if (this.items.length <= 1 || !this.items) return false
 
       return [...this.items].reverse().slice(1)
     },
-    $_ocBreadcrumb_currentFolder() {
+    currentFolder() {
       if (this.items.length === 0 || !this.items) return false
 
       return [...this.items].reverse()[0]
     },
+    homeAccessibilityLabelValue() {
+      return this.homeAccessibilityLabel || this.$gettext("Go to Home")
```

Comment:
I don't know if it's too techie, but the breadcrumbs are used in others contexts as well, so `Go to root directory` would be more appropriate. But again, I have the feeling that that's too techie... what do you think?

Reply:
I had the exact same thought. But decided against it because then we presume that home always links to the filelist. Otherwise "root directory" wouldn't make sense. If we can happily make that presumption, I find `Go to root directory` totally fine 🙂

Reply:
fine by me 👍